frontend/src/components/Topbar/Topbar.jsx

```
@@ -0,0 +1,33 @@
+export default function Topbar() {
+  return (
+    <div className="flex items-center h-14 px-6 border-b border-[#222] bg-[#232b3e] sticky top-0 z-30">
+      <span className="text-gray-300 text-sm flex items-center gap-2">
+        <span className="font-semibold">Team Space</span>
+        <span>/</span>
+        <span>Projects</span>
+        <span>/</span>
+        <span className="font-semibold">Project 1</span>
+        <FaChevronDown className="ml-2 text-xs" />
+      </span>
+      <button className="ml-6 text-gray-400 hover:text-yellow-400 flex items-center gap-1">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)